tests/plugins/add-to-triage-project/index.js

```diff
@@ -0,0 +1,64 @@
+"use strict";
+
+const { addToTriageProject } = require("../../../src/plugins/index");
+const nock = require("nock");
+const { Application } = require("probot");
+const { NEEDS_TRIAGE_COLUMN_ID } = require("../../../src/constants");
+const GitHubApi = require("@octokit/rest");
```

Comment:
nits: just a style preference: we might want a rule to sort the requires to:
```js
const nock = require("nock");
const GitHubApi = require("@octokit/rest");
const { Application } = require("probot");

const { addToTriageProject } = require("../../../src/plugins/index");
const { NEEDS_TRIAGE_COLUMN_ID } = require("../../../src/constants");
```

Reply:
@aladdin-add we try not to nitpick on style in pull requests.